pkg/names/names_test.go

```diff
@@ -0,0 +1,76 @@
+package names
+
+import (
+	"os"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("IncludeRelationshipLabels", func() {
+	Context("When env vars don't exist", func() {
```

Comment:
we should also include the case were the labels are already set (upgrade scenario)